FlappyBird/Bird.java

```diff
@@ -0,0 +1,67 @@
+package FlappyBird;
```

Comment:
jak chcesz zeby ktos Ci sprawdzal kod, to wystawiaj pull requesty. Robisz tak masz jedna gałąź master i wrzucasz tam tylko jakis poczatkowy stan projektu, potem robisz nowy branch i dokladasz kolejny kod czyli jakies nowe funkcjonalnosci albo fixy bugow, jak juz skonczysz wystawiasz robisz ``commit``, ``push`` i na stronie wystawiasz ``pull request`` z twojej nowej galezi do ``master`` i dodajesz mnie albo kogos do ``review``, a tak musialem sam forkowac twoje repo, usuwac to i potem robic cherryPicki Twoich commitow :P